whatsapp_archive.py

```diff
@@ -15,7 +15,7 @@
 # future and so this application will need to be updated.
 DATE_RE = '(?P<date>[\d/-]+)'
 TIME_RE = '(?P<time>[\d:]+( [AP]M)?)'
-DATETIME_RE = '\[?' + DATE_RE + ',? ' + TIME_RE + '\]?'
+DATETIME_RE = '\[?' + DATE_RE + ',? ' + 'à? ?' + TIME_RE + '\]?'
```

Comment:
Can you give it a name and add as a variable in the line above? Say, "AT_RE" or something similar. (If I understand the meaning correctly.)